plugins/graphiql/package.json

```diff
@@ -60,5 +60,22 @@
   },
   "files": [
     "dist"
-  ]
+  ],
```

Comment:
Never worked with codemods on my own, but something like that might be nice here too 🤔

Reply:
It seems likely that we'll have to use codemods for some of the complex installation steps :D

Reply:
Yep, prolly something like `ts-morph` for this. We opted for keeping it simple for now though